distribution/docs/src/main/resources/_contents/_plugins/historian-policy-plugin-contents.adoc

```diff
@@ -0,0 +1,19 @@
+
+==== Historian Plugin
```

Comment:
@rzwiefel do you think we should remove this "historian plugin" section from the docs?